README.md

```diff
@@ -0,0 +1,168 @@
+# ROCM-CORE Introduction
+
+ROCM-CORE is a package which can be used to get ROCm release version, get ROCm install path information etc.
+It is also important to note that ROCM-CORE takes the role as a base component on which all of ROCm can depend,
+to make it easy to remove all of ROCm with a package manager.
+
+getROCmVersion function provides the ROCm version. 
+
+It also provides an example Lmod modules files for the ROCm release.
+
+Lmod module files can be loaded with the following commands.
+``` shell
+module load rocm/x.y or 
+module load rocm
+```
+
+You can find sources and binaries in our [GitHub repository](https://github.com/ROCm/rocm-core).
+
+> [!NOTE]
+> As with all ROCm projects, the documentation is open source. For more information, see [Contribute to ROCm documentation](https://rocm.docs.amd.com/en/latest/contribute/contributing.html).
+
+## Installing ROCM-CORE
+
+Before we proceed with how to install, take a look on references given below  to understand System requirements, ROCm Installation prerequisites, ROCm package repository set up etc.
```

Comment:
This sentence says "look below for some more info, then come back here" but there is nothing to come back to! I would delete the entire line I think.

Reply:
This is referring to the section below with prerequisites, in this version this is not modified